pkgs/tools/system/at/default.nix

```diff
@@ -5,22 +5,38 @@
 
 stdenv.mkDerivation rec {
   pname = "at";
-  version = "3.1.23";
+  version = "3.2.5";
 
   src = fetchurl {
     # Debian is apparently the last location where it can be found.
     url = "mirror://debian/pool/main/a/at/at_${version}.orig.tar.gz";
-    sha256 = "040pr2ivfbrhvrhzis97cpwfkzpr7nin33nc301aga5aajlhlicp";
+    hash = "sha256-uwZrOJ18m7nYSjVzgDK4XDDLp9lJ91gZKtxyyUd/07g=";
   };
 
   patches = [
-    ./install.patch
+    # Remove glibc assumption
     (fetchpatch {
       url = "https://raw.githubusercontent.com/riscv/riscv-poky/master/meta/recipes-extended/at/at/0001-remove-glibc-assumption.patch";
-      sha256 = "1rk4hskp0c1jqkanzdxf873i6jgki3xhrm609fsam8an8sl1njnm";
+      hash = "sha256-1UobqEZWoaq0S8DUDPuI80kTx0Gut2/VxDIwcKeGZOY=";
     })
   ];
 
+  postPatch = ''
+    # Remove chown commands and setuid bit
+    substituteInPlace Makefile.in \
+      --replace '$(INSTALL) -o root -g root' '$(INSTALL)' \
+      --replace '$(INSTALL) -d -o root -g root' '$(INSTALL) -d' \
+      --replace '$(INSTALL) -g root -o root' '$(INSTALL)' \
+      --replace '$(INSTALL) -g $(DAEMON_GROUPNAME) -o $(DAEMON_USERNAME)' '$(INSTALL)' \
+      --replace '$(INSTALL) -o root -g $(DAEMON_GROUPNAME)' '$(INSTALL)' \
```

Comment:
```suggestion
      --replace ' -o root ' ' ' \
      --replace ' -g root ' ' ' \
      --replace ' -o $(DAEMON_GROUPNAME) ' ' ' \
      --replace ' -g $(DAEMON_GROUPNAME) ' ' ' \
```

Reply:
Hello. I've done the changes, thanks for the review!